linkerd/identity/src/metrics.rs

```diff
@@ -37,6 +37,14 @@ impl CertMetrics {
             refresh_ts.clone(),
         );
 
+        let clock_time_ts = prom::Gauge::<f64, ClockMetric>::default();
```

Comment:
I think we probably aught to move this gauge into the process metrics so that it's more clearly reusable across all metrics exported by the process -- there's nothing really identity-specific about it :)

Reply:
I agree that makes more sense for a general gauge like this, updated.